### DISCUSSION TITLE
Need help with environmental variables

### DISCUSSION THREAD

**Discussion by Jamiecharris** [vercel/turborepo#2613] [Help] [answered]
I've defined the envs in the turbo.json but still getting this error ![image](https://user-images.githubusercontent.com/55756319/200312548-8b39fa36-2ffc-43e1-b967-1e73d3e3b63f.png)

**Comment by anthonyshew** [comment, marked as answer]
Going to close this as stale. If you're still having a problem, please open an issue!
